tq/transfer_queue.go

```diff
@@ -344,14 +344,14 @@ func (q *TransferQueue) enqueueAndCollectRetriesFor(batch batch) (batch, error)
 					tracerx.Printf("tq: enqueue retry #%d for %q (size: %d)", count, tr.Oid, tr.Size)
 					next = append(next, t)
 				} else {
-					if !IsActionMissingError(err) {
-						q.errorc <- errors.Errorf("[%v] %v", tr.Name, err)
-					}
+					q.errorc <- errors.Errorf("[%v] %v", tr.Name, err)
```

Comment:
Good question, but I don't think that this is the behavior that's exhibited. A missing action now returns `(nil, nil)`, which would skip this block entirely, and only execute:

```go
} else if a == nil {
      q.Skip(o.Size)
      q.wait.Done()
}
```

I should've discussed this change more in the original post, but when I implemented the `multiActionSet`, I found juggling the `IsActionMissingErr` calls to be awkward, and it made sense to just remove it entirely, since "no action, no error" corresponds to a `(nil, nil)` return value well.